src/main/java/io/sentry/cli/SentryCliRunner.java

```diff
@@ -65,16 +65,18 @@ public SentryCliRunner(
                       attributes(
                           attribute("executable", executable),
                           attribute("failOnError", String.valueOf(failOnError)),
-                          attribute("output", logFile.getAbsolutePath())),
+                          attribute("output", escape(logFile.getAbsolutePath(), isWindows))),
                       element(name("arg"), attributes(attribute("value", cArg))),
                       element(
                           name("arg"),
                           attributes(
                               attribute(
                                   "value",
-                                  getCliPath(mavenProject, sentryCliExecutablePath)
-                                      + " "
-                                      + sentryCliCommand)))))),
+                                  escape(
+                                      getCliPath(mavenProject, sentryCliExecutablePath)
+                                          + " "
+                                          + sentryCliCommand,
```

Comment:
escape should only cover `getCliPath`.
Otherwise all spaces within the `sentryCliCommand` will be escaped, resulting in an invalid command string for the sentry cli

Reply:
Whoever sends in the `sentryCliCommand` needs to take care of escaping the spaces in the path.
E.g. the `UploadSourceBundleMojo` for that the `escape` method could be public and `isWindows` could be a field of `SentryCliRunner`, so that the `escape` method does not need it as a parameter

Reply:
So, in `UploadSourceBundleMojo` line 145 could be changed to:
`bundleSourcesCommand.add(cliRunner.escape(sourceRoot));`